docker/production.dockerfile

```diff
@@ -12,41 +12,35 @@
 # docker buildx inspect --bootstrap
 # docker buildx build -f docker/production.dockerfile --platform linux/amd64,linux/arm64 --no-cache --push -t zavy86/wikidocs .
 #
+# Docker compose rebuild
+# docker compose -f docker-compose.prod.yml down && docker compose -f docker-compose.prod.yml up --build -d
+#
 
-FROM alpine:3.15
+FROM alpine:3.20
 
 ARG DEPENDENCIES="\
-apache2 \
-php7 \
-php7-apache2 \
-php7-dom \
-php7-json \
-php7-mbstring \
-php7-session \
-php7-xml \
 shadow \
+apache2 \
+php \
+php-apache2 \
+php-dom \
+php-json \
+php-mbstring \
+php-session \
+php-xml \
 "
 
 # installation
 RUN apk add --no-cache $DEPENDENCIES
 
-# configure apache
+# configure httpd
 RUN sed -ri \
     -e 's!^#(LoadModule rewrite_module .*)$!\1!g' \
     -e 's!^(\s*AllowOverride) None.*$!\1 All!g' \
     "/etc/apache2/httpd.conf"
 RUN echo "ServerName localhost" >> /etc/apache2/httpd.conf
 RUN rm /var/www/localhost/htdocs/index.html
 
-# copy application files
-COPY . /var/www/localhost/htdocs/
-
-# make a link for datasets volume
-RUN ln -s /var/www/localhost/htdocs/datasets /
-
-# copy htaccess files from samples
-COPY ./sample.htaccess /var/www/localhost/htdocs/.htaccess
-
```

Comment:
questa parte va ripristinata, serve per generare l'immagine da inviare all'hub, la gente se lo scarica da li non ha il sorgente in locale, quindi va incluso..